src/components/ImageSocialShare.vue

```diff
@@ -31,6 +31,9 @@ export default {
     imageURL() {
       return this.image.foreign_landing_url;
     },
+    image_URL() {
```

Comment:
- The naming of this prop could be confusing as it's named very similarly to `imageURL` above, but they return different things.
- This prop isn't used anywhere in the `ImageSocialShare` component, maybe you meant to add it to `SocialShareButtons` instead?